components.html

```diff
@@ -36,6 +36,17 @@ <h2 class="component__name">Logo</h2>
               </a>
             </div>
           </li>
+          <li class="component">
+            <h2 class="component__name">Section Intro</h2>
+            <p class="component__description">Block with text in the center on a white background</p>
```

Comment:
Hi again! Thank you for your changes! One more suggestion: Please, add a more detailed description. You should explain all classes that you can use for this component. You might use this description as an example: 
![image](https://github.com/p-foundation/pink-app/assets/126704357/11aaa42e-55ab-476f-bdb6-2dba129e1431)
Also, add this description to the description of this pull request.